setup.py

```diff
@@ -89,6 +89,7 @@
     keywords="qiskit sdk quantum",
     packages=find_packages(exclude=['test*']),
     install_requires=REQUIREMENTS,
+    setup_requires=['Cython>=0.29.5'],
```

Comment:
Do we really need that high a version here?  I think `0.27.1` or higher should work.

Reply:
Sure, we can change it. I just picked the version I had installed locally when I was testing which I guess was the latest.

Reply:
ok cool. And is it 'Cython' and not 'cython'?

Reply:
The package on pypi is `Cython`: https://pypi.org/project/Cython/ although pip is case insensitive so it doesn't really matter.

Reply:
Ok, just double checking.  It is `cython` in conda.